scipy/stats/tests/test_distributions.py

```diff
@@ -371,6 +371,51 @@ def test_pdf_zero(self):
         assert_equal(stats.geninvgauss.pdf(2e6, 50, 2), 0)
 
 
+class TestGenHyperbolic(object):
```

Comment:
I like that you test against special case distributions. Is there a reference for values that do not correspond with special cases? For instance, are there published values of the PDF somewhere, or is there another implementation you can compare against?

Reply:
Compared pdf cdf and stats against existing R implementation